### ISSUE TITLE
Prohibiting use of same tracking number for home mouthwash kits (Biospecimen Dashboard)

### ISSUE THREAD

**Issue by erincschwartz** [episphere/connect#1038]
Currently, the Dashboard is allowing the same tracking number to be entered for both the Return Kit Tracking Number (on Assemble Kit screen) and the Supply Kit Tracking Number (on Assign Kit screen), for both FedEx and USPS tracking numbers.

The system is also allowing more than one kit to be assembled with the same Return Kit Tracking Number, see below: 

![Image](https://github.com/episphere/connect/assets/157617486/d8016c0b-9cf9-4d59-a555-bcb03547be46)

Once a tracking number is used for either Return Kit Tracking Number or Supply Kit Tracking Number, the system should **not** allow it to be used again in any tracking number field. If FNL enters a tracking number that has already been used, an error message should appear to indicate "This tracking number has already been used"

**Comment by amber-emmes** [comment]
This has been merged into dev and is ready for dev testing.

**Comment by erincschwartz** [comment]
@amber-emmes In Dev, the functionality is working well. Can you update the error message on the Assign Kits page to resemble the error that appears on the Kit Assembly page? It is much more clear to understand the problem with the message "This tracking number has already been used" than the generic error message that is currently popping up. 

Error message on Assign Kits page: 

![Image](https://github.com/user-attachments/assets/00fae804-e5a7-4920-88dc-1d33b3474578)

Error message on Kit Assembly page:

![Image](https://github.com/user-attachments/assets/f206f621-87bc-4e98-9e5f-73a24c567acd)

**Comment by amber-emmes** [comment]
The short answer is that "Error while assigning a kit" is the message for all error messages on this page, which is something that I definitely think should be addressed but which I think is its own ticket for a future release due to the fact that this involves changing the workflow of the entire page and corresponding backend process such that more useful error messages can be passed back in the first place.

**Comment by erincschwartz** [comment]
Ok, just confirming, for this release, we cannot make the error message change on the Assign Kit page?

**Comment by amber-emmes** [comment]
I don't think that would be advisable in the given timeframe before dev freeze, no.

**Comment by erincschwartz** [comment]
Ok, then this is ready for Stage. I will create new issue for refining the error messaging in August/September, as time permits.

**Comment by erincschwartz** [comment]
Tested in Stage, ready for Prod

**Comment by sonyekere** [comment]
In production.
